src/main/kotlin/hu/kirdev/schpincer/service/OrderService.kt

```diff
@@ -17,13 +18,14 @@ import java.io.IOException
 import java.util.*
 import javax.servlet.http.HttpServletRequest
 
-
 private val List<OrderEntity>.highestPriority: Int
     get() = this.maxBy { it.priority }?.priority ?: 1
 
-const val ORDER_ABSOLUTE = "absolute"
-const val ORDER_GROUPED = "grouped"
-const val ORDER_ROOMS = "rooms"
+enum class OrderStrategy(val representation: String) {
+    ORDER_ABSOLUTE("absolute"),
+    ORDER_GROUPED("grouped"),
+    ORDER_ROOMS("rooms");
+}
```

Comment:
This is nice.
templates/openingShow.html from line 243 depends on these constants, so let's change the options' th:value to `${T(hu.kirdev.schpincer.service.OrderStrategy).ORDER_ABSOLUTE.representation}` and so on.